### DISCUSSION TITLE
Frequently times out when uploading NPM version packages

### DISCUSSION THREAD

**Discussion by liukkm** [community/community#78731] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Last Friday, I uploaded the new version package to the npm official website warehouse for uploading. When I uploaded the new version package from Monday, I was informed of the error. The reason for the error is as follows: connect ETIMEDOUT 104.16.26.34:443, so far I have not found a solution to this problem, in addition, other computers in our company still encounter the same problem, the variable exclusion method should not be a problem of computer network configuration, I guess there are two reasons: 1. 2. The connection to the NPM official library failed. I wonder if there are others who have encountered the same problem as me. I look forward to your early reply.

**Comment by rockfang** [comment]
the same problem

**Comment by rockfang** [comment, marked as answer]
@liukkm I solve this temporarily by set npm proxy.
1. start my proxy soft.
2. set to the npm proxy `npm config set proxy http://127.0.0.1:7890/`

**Reply by liukkm** [reply]
Thank you for your solution, I just tried it and found it was successful, maybe the VPN tampered with the configuration of my local computer.
